code/datums/wires/wires.dm

```diff
@@ -256,11 +260,10 @@ var/const/POWER = 8
 
 
 /datum/wires/proc/Pulse(var/obj/item/device/assembly/signaler/S)
-
-
 	for(var/colour in signallers)
 		if(S == signallers[colour])
 			PulseColour(colour)
+			log_attack("The [GetWireName(wires[colour])] wire of airlock named \"[holder]\" at [holder.x],[holder.y],[holder.z] has been pulsed with [S], activated by [key_name(usr)].")
```

Comment:
Not 100% sure about the use of "usr" here, but it seemed to work perfectly well, even getting the correct mob when activated with a voice signaler and such